app/gig.py

```diff
@@ -1,19 +1,37 @@
 import os
 import shutil
 import pkg_resources
+import click
 
 
 from .map import MAP
 
 
 class GIG(object):
+    __dest = os.getcwd() + '/.gitignore'
```

Comment:
Why don't we just move this inside the `__init__`, it would be better since this is empty. And also no need for double underscores for private members single are better for readability and also follow the convention. :+1: